### PR TITLE
[server] install job for orgonly migration

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

Migrate users in a job

Depends on #17672

## Related Issue(s)

Fixes WEB-131

#### Preview status

gitpod:summary

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [x] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold

### CODE REVIEW

File: components/server/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "env": {
+                "DB_PORT": "23306",
+                "DB_ENCRYPTION_KEYS": "[{\"name\":\"general\",\"version\":1,\"primary\":true,\"material\":\"5vRrp0H4oRgdkPnX1qQcS54Q0xggr6iyho42IQ1rO+c=\"}]"
+            },
+            "name": "Run Mocha Test in Editor",
```
Comment (selfcontained) [inline components/server/.vscode/launch.json:11]: ❤️

**Review by selfcontained — COMMENTED** [review]
I'm not super familiar w/ our Jobs setup, but this makes sense to me. It would run every 5 minutes, and migrate 100 users. I wasn't familiar with that query builder syntax to query against json objects. Do you know how that works? I wasn't able to find any docs about that.

**Comment by svenefftinge** [comment]
>  I wasn't familiar with that query builder syntax to query against json objects. 

It directly translates to MySQL which has special operators and functions for JSON ([docs](https://dev.mysql.com/doc/refman/5.7/en/json.html)).

**Review by selfcontained — APPROVED** [review]

**Pull request merged**